src/ScottPlot5/ScottPlot5/Plottables/Tooltip.cs

```diff
@@ -0,0 +1,98 @@
+namespace ScottPlot.Plottables;
+
+/// <summary>
+/// A tooltip displays a text bubble pointing to a specific location in X/Y space.
+/// The position of the bubble moves according to the axis limits to best display the text in the data area.
+/// </summary>
+public class Tooltip : Text, IPlottable, IHasLine, IHasFill
```

Comment:
I'm a little conflicted on inheriting from `Text` here, since `Text` inherits from `LabelStyleProperties` and I don't think we respect things like `LabelBackgroundColor`, and if they differ I think we get the label drawing overtop of us with a different fill colour.

I think this would be alleviated if we didn't define our own FillStyle, but then we wouldn't support things like hatched fills (which honestly is probably fine). I think it's definitely simpler if we have the tooltip be separate and then it will only have fields that behave in a consistent manner with eachother